### DISCUSSION TITLE
Is there a way to access the documentation for indyproject?

### DISCUSSION THREAD

**Discussion by AnthonySWest** [IndySockets/Indy#612] [Q&A]
I am trying to download the documentation from page:

indyproject.org/documentation/

Clicking on any of the documentation links results in the following error (in FireFox):

```
Hmm. We’re having trouble finding that site.

We can’t connect to the server at ww2.indyproject.org.

If you entered the right address, you can:

    Try again later
    Check your network connection
    Check that Firefox has permission to access the web (you might be connected but behind a firewall)
```

Bad link, or site down? I have tried a few times throughout this week.

**Comment by rlebeau** [comment]
Please read: https://www.indyproject.org/2021/02/10/links-to-old-indy-website-pages-are-currently-broken/

**Comment by rlebeau** [comment]
Duplicate of #515

**Comment by jachguate** [comment]
I have uploaded a copy I found locally of the documentation to github at
https://github.com/jachguate/indydocs

The same documentation is accessible online here:
https://jachguate.github.io/indydocs/html/

Juan Antonio Castillo H. (jachguate)
Blog: http://jachguate.wordpress.com

**Comment by AnthonySWest** [comment]
Thanks!
